server/personality/personality.service.ts

```diff
@@ -213,7 +213,7 @@ export class PersonalityService {
 
         try {
             // This line may cause a false positive in sonarCloud because if we remove the await, we cannot iterate through the results
-            const nameSpace = this.req.params.namespace;
+            const nameSpace = this.req.params.namespace || NameSpaceEnum.Main;
```

Comment:
praise: Nice work. We still need to filter the claims by nameSpace in the `/debate` page